java/jdbc-relational-duality-example/src/main/java/org/oracle/Main.java

```diff
@@ -0,0 +1,128 @@
+/* Copyright (c) 2021, 2022, Oracle and/or its affiliates.
+This software is dual-licensed to you under the Universal Permissive License
+(UPL) 1.0 as shown at https://oss.oracle.com/licenses/upl or Apache License
+2.0 as shown at http://www.apache.org/licenses/LICENSE-2.0. You may choose
+either license.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+DESCRIPTION
+Main - Runner class calling multiple methods
+*/
+
+package org.oracle;
+
+import oracle.sql.json.OracleJsonArray;
+import oracle.sql.json.OracleJsonFactory;
+import oracle.sql.json.OracleJsonObject;
+
+
+public class Main {
+	private static DatabaseConfig pds;
```

Comment:
Please use the same indentation in all your Java files to make the reading experience easier. For example, you can use 2 space characters.

Reply:
OK I have fixed it to 2 spaces